src/components/FoodPantryCard/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FunctionComponent } from 'react';
+import { FoodPantry } from 'utils/types';
+
+interface Props {
+  pantry: FoodPantry;
+}
+
+const FoodPantryCard: FunctionComponent<Props> = ({ pantry }) => {
+  return (
+    <div>
+      <h1 id="pantry_name">Name: {pantry.name}</h1>
```

Comment:
I noticed that there is an option for a website types.ts. Could look at possibly adding that to the card as well.